lua/gatekeeper/policylib.lua

```diff
@@ -281,3 +281,37 @@ function decision_web(policy, tx_rate_kib_sec, cap_expire_sec,
 		policy, tx_rate_kib_sec, tx_rate_kib_sec * 0.05, -- 5%
 		cap_expire_sec, next_renewal_ms, renewal_step_ms, false)
 end
+
+local function lpm_add_tbl8s(ip_addr, prefix_len, prefixes)
+	local prefix
+	local masked_ip
+
+	if prefix_len <= 24 then
+		return prefixes
+	end
+
+	masked_ip = lpmlib.ip_mask_addr(ip_addr, 24)
+	prefix = masked_ip .. "/24"
```

Comment:
All prefixes in table `prefixes` are /24, so we don't need to concatenate the length here.